e2e/shortcut-action/shortcut-action.po.ts

```diff
@@ -0,0 +1,15 @@
+import { element, by } from 'protractor/globals';
+import { Ng2JsonEditorPage } from '../app.po';
+
+export class ShortcutActionPage extends Ng2JsonEditorPage {
+
+  getNumberOfTextareaElementsInInnerTable(id: string) {
+    let tableElem = element(by.id(id));
+    return tableElem.all(by.css('tr[id*=\'' + id + '\']')).count();
+  }
+
+  getNumberOfTextareaElementsInRootTable(id: string) {
+    let tableElem = element(by.id(id));
+    return tableElem.all(by.css('div+div')).count();
```

Comment:
Instead of `div+div` you can count the number of `table` elements with id like the parameter.